cmd/apiregister-gen/generators/versioned_generator.go

```diff
@@ -79,7 +91,9 @@ var (
 		{{ range $subresource := $api.Subresources -}}
 		builders.NewApiResourceWithStorage(
 			{{ $api.Group }}.Internal{{ $subresource.REST }},
+			builders.SchemeFnsSingleton,
 			func() runtime.Object { return &{{ $subresource.Request }}{} }, // Register versioned resource
+			nil,
 			&{{ $api.Group }}.{{ $subresource.REST }}{ {{$api.Group}}.New{{$api.Kind}}Registry({{$api.Group}}{{$api.Kind}}Storage) },
```

Comment:
If the resource defines its own REST, we want to follow this pattern, not the storage patterns above.